build/get-babel-config.js

```diff
@@ -146,6 +146,7 @@ function fusionPreset(
       require.resolve('./babel-plugins/babel-plugin-sw'),
       require.resolve('./babel-plugins/babel-plugin-sync-chunk-paths'),
       require.resolve('./babel-plugins/babel-plugin-chunkid'),
+      require.resolve('babel-plugin-transform-styletron-display-name'),
```

Comment:
Thanks for the PR! Wondering if there is two options that we could potentially do here:

1 - Ship this babel transform as part of the scaffold, included in the .fusionrc file.
2 - Ship this as a compiler plugin in fusion-plugin-styletron-react.

Just thinking that we should decouple concerns here if possible. Any thoughts @rtsao?

Reply:
thanks for the review folks. I honestly need to get a better sense of the fusion plugin patterns.

FYI I added a new pr to the plugin [here](https://github.com/styletron/styletron/pull/290) so that it solves my original problem.